tests/testrom.c

```diff
@@ -202,12 +203,11 @@ static const struct Testsuite
 	TEST_FUNC(test_eepromfs,                   0, TEST_FLAGS_IO),
 	TEST_FUNC(test_cache_invalidate,    	1763, TEST_FLAGS_NONE),
 	TEST_FUNC(test_debug_sdfs,             	   0, TEST_FLAGS_NO_BENCHMARK),
-	TEST_FUNC(test_dma_read_misalign,       7003, TEST_FLAGS_NONE),
+	TEST_FUNC(test_dma_read_misalign,       7003, TEST_FLAGS_NO_BENCHMARK),
```

Comment:
Can you please leave this with benchmark? It doesn't fail on many consoles so there must be something else that we should look into. If you change this, we will simply forget about it.